server/src/main/java/com/linecorp/centraldogma/server/Main.java

```diff
@@ -199,35 +207,60 @@ public void run() {
             }
         });
 
-        // Initialize the main with a dummy context.
-        main.init(new DaemonContextImpl(args));
-
         // Exit if initialization failed.
         if (main.state != State.INITIALIZED) {
             System.exit(1);
             return;
         }
 
-        main.start();
+        try {
+            main.start();
+        } catch (Throwable cause) {
+            logger.warn("Failed to start the Central Dogma:", cause);
```

Comment:
I guess this should be logged at `ERROR` level because it will prevent Central Dogma from working at all. (Irrecoverable)